packages/js/src/ui/index.css

```diff
@@ -0,0 +1,469 @@
+/*
```

Comment:
✏ **suggestion:** Please prettify this file because the code and comments look odd